aten/src/ATen/core/op_registration/op_registration.h

```diff
@@ -605,23 +605,22 @@ class CAFFE2_API RegisterOperators final {
 //
 // A quick tour of a few usage examples:
 //
-//  auto register = torch::import()
-//
+//  // Specify a namespace.  This namespace must not be used by any
```

Comment:
bikeshed: "...and populate it with all its operator definitions. (Implementations can be added here or separately.)"